CleverTapSDK/CTUserInfoMigrator.m

```diff
@@ -1,26 +1,67 @@
 #import "CTUserInfoMigrator.h"
 #import "CTConstants.h"
+#import "CTPreferences.h"
 
 @implementation CTUserInfoMigrator
 
-+ (void)migrateUserInfoFileForAccountID:(NSString *)acc_id deviceID:(NSString *)device_id {
++ (void)migrateUserInfoFileForAccountID:(NSString *)acc_id deviceID:(NSString *)device_id config:(CleverTapInstanceConfig *) config {
```

Comment:
Since you pass the whole `CleverTapInstanceConfig` instance, you can read the `accountId` from the config, instead of passing it as another parameter.

Reply:
Done the changes